docs/DeveloperGuide.md

```diff
@@ -158,21 +158,51 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Lesson Time Parameter
+This parameter allows users to keep track of a student's lesson timings. Multiple lesson times can be added for a single student.
+
+#### Design Considerations
+**Treatment of clashing timings**
+
+- **Current Implementation (Alternative 1):**
+    - **Description**: Clashing lesson timings across and within students are not rejected.
```

Comment:
Clashing lesson timings are permitted across all students and within each student's schedule.

Think this sounds better